pkg/client/results/junit.go

```diff
@@ -174,9 +174,12 @@ func junitProcessFile(pluginDir, currentFile string) (Item, error) {
 	}
 
 	resultObj := Item{
-		Name:     filepath.Base(currentFile),
-		Status:   StatusUnknown,
-		Metadata: map[string]string{"file": relPath},
+		Name:   filepath.Base(currentFile),
+		Status: StatusUnknown,
+		Metadata: map[string]string{
+			metadataFileKey: relPath,
```

Comment:
Is it necessary to have the file key/entry here given that it’s already in the name? Or is it to just to make it really clear?

Reply:
Yes, they are unique since the `Name` is just the basename and the actual `Metadata: {"file":<val>}` will be the full filename relative to where the sonobuoy_results.yaml file is located at.

For instance:
```
  - name: systemd_logs
    status: passed
    meta:
      file: results/kind-control-plane/systemd_logs
```